### ISSUE TITLE
Screen reader support needed in more options menu

### ISSUE THREAD

**Issue by ilayaraja97** [mozilla-mobile/focus-android#3149]
## Steps to reproduce
0. Turn on TalkBack (accessibility feature in android)
1. Open Firefox Focus
2. Open any webpage
3. Open the menu
4. Hover over each menu item

### Expected behavior
TalkBack oration should be more descriptive to help those who are new to phones and UI.
_Find in Page_ must say "Find in page, search for the term within the page"
_Settings_ should say "Settings menu" (related #2947) 

### Actual behavior
_Find in Page_ says "Find in Page"
_Settings_ says "Settings"

### Device information

* Android device: Huawei LLD-AL10
* Focus version: 7.0 (Build #11)

**Comment by bbinto** [comment]
@BrianNJones and @brampitoyo: would you mind consulting with each other and proposing / finalizing the new strings to be used.

**Comment by brampitoyo** [comment]
@ilayaraja97 Before I talk to @BrianNJones, I’d like to schedule a chat with you and talk through this. When will you be available?

Have you heard of how TalkBack describes menu items in other apps? I would encourage surveying them and reporting your findings here.

I’m open to possibly adding a subtitle – as long as it’s invisible – under each menu item. But our general principle shouldn’t be to write supplementary descriptions when it’s not absolutely necessary. Instead, we should write each menu item so they’re as short and understandable as possible.

**Comment by ilayaraja97** [comment]
@brampitoyo good idea. I'll look into the behavior of other apps and report it here.

**Comment by ekager** [comment]
@brampitoyo we can add a content description that could add additional information without changing the UI

**Comment by brampitoyo** [comment]
@ekager This sounds like a good idea. For our deliverables, do we need to write all the subtitles to each menu item (as well as other pages and dialogues)?

-----

Speaking of dialogues, TPG had a feedback that may be related to this issue (if unrelated, we can open a new issue and solve it there).

<img width="420" src="https://user-images.githubusercontent.com/51007/45459794-1efc8d80-b74e-11e8-8666-3d1faa5fe79c.png">

![screenshot_2018-09-13 firefox focus in-app communications comment mode - invision 1](https://user-images.githubusercontent.com/51007/45459815-389dd500-b74e-11e8-8a19-d3a3b0b44953.png)

**Comment by sblatz** [comment]
@brampitoyo my understanding is that this ticket is specifically for the menu options:
* Trackers blocked
* Share
* Add to home screen
* Find in page
* Open in...
* Request Desktop Site
* Settings
* What's new
* Help

Trying to add more content descriptions universally throughout the app would be helpful, but is likely too daunting to handle in one ticket. Once you decide on strings I'd be happy to tackle this :)

Regarding @brampitoyo's comment about using more native UI elements to get accessibility descriptions for free--this is a great idea, but would not help a whole lot in this case as we would still have to describe what the menu items do. As far as I'm aware, we *do* use mostly native UI elements as is.
